kong/db/declarative/init.lua

```diff
@@ -92,17 +105,16 @@ function Config:parse_string(contents, filename, accept, old_hash)
   accept = accept or { yaml = true, json = true }
 
   local dc_table, err
-  if accept.yaml and ((not filename) or filename:match("ya?ml$")) then
+  if (accept.yaml and ((not filename) or filename:match("ya?ml$")))
+  or (accept.json and filename:match("json$"))
+  then
     local pok
     pok, dc_table, err = pcall(lyaml.load, contents)
     if not pok then
       err = dc_table
       dc_table = nil
     end
 
-  elseif accept.json and filename:match("json$") then
-    dc_table, err = cjson.decode(contents)
```

Comment:
This is going to cause `.json` input files to contain arbitrary YAML, which is a much larger format. The `/config` endpoint doesn't do a very good job in making a distinction, but `db_import` does... I would keep this around, but not a blocker

Reply:
(also, cjson is probably faster in case this ever matters)

Reply:
Ok, let's remove it for now. Removed. :-).